keyboards/binepad/candypad/candypad_oled.c

```diff
@@ -0,0 +1,244 @@
+// Copyright 2023 binepad (@binepad)
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Portions of this code are based on [Andrew Kannan](https://github.com/awkannan1)'s
+//  work on the Satisfaction75
+
+#include "candypad.h"
+#include "keymap_introspection.h"
```

Comment:
This shouldn't be needed.

Reply:
- `keymap_introspection.h` is needed for the `keycode_at_encodermap_location` function since this LED displays the mode the encoders are in.
- Have removed the `candypad.h` file tho.